.github/workflows/ci.yml

```diff
@@ -0,0 +1,238 @@
+      BACKEND: postgres
+      POSTGRES_VERSION: 9.6
+      PYTHON_VERSION: 3.6
+    steps:
+      - uses: actions/checkout@master
+      - name: "Tests [Postgres9.6][Py3.6][core]"
+        run: ./scripts/ci/ci_run_airflow_testing.sh --ignore=tests/providers
+
+
+  tests-p37-sqlite-integrations:
+    name: "Tests [Sqlite][3.7][integrations]"
+    runs-on: ubuntu-latest
+    needs: [statics, statics-tests]
+    env:
+      TRAVIS_JOB_NAME: "Tests [Sqlite][3.7][integrations]"
+      BACKEND: sqlite
+      PYTHON_VERSION: 3.7
+      ENABLED_INTEGRATIONS: "cassandra kerberos mongo openldap rabbitmq redis"
+      RUN_INTEGRATION_TESTS: all
+    steps:
+      - uses: actions/checkout@master
+      - name: "Tests [Sqlite][3.7][integrations]"
+        run: ./scripts/ci/ci_run_airflow_testing.sh
+
+  tests-p36-sqlite:
+    name: "Tests [Sqlite][Py3.6]"
+    runs-on: ubuntu-latest
+    needs: [statics, statics-tests]
+    env:
+      TRAVIS_JOB_NAME: "Tests [Sqlite][Py3.6]"
+      BACKEND: sqlite
+      PYTHON_VERSION: 3.6
+    steps:
+      - uses: actions/checkout@master
+      - name: "Tests [Sqlite][Py3.6]"
+        run: ./scripts/ci/ci_run_airflow_testing.sh
+
+  tests-p36-mysql-integrations:
+    name: "Tests [MySQL][Py3.6][integrations]"
+    runs-on: ubuntu-latest
+    needs: [statics, statics-tests]
+    env:
+      TRAVIS_JOB_NAME: "Tests [MySQL][Py3.6][integrations]"
+      BACKEND: sqlite
+      PYTHON_VERSION: 3.6
+      MYSQL_VERSION: 5.7
+      ENABLED_INTEGRATIONS: "cassandra kerberos mongo openldap rabbitmq redis"
+      RUN_INTEGRATION_TESTS: all
+    steps:
+      - uses: actions/checkout@master
+      - name: "Tests [MySQL][Py3.6][integrations]"
+        run: ./scripts/ci/ci_run_airflow_testing.sh
+
+  tests-p36-mysql-providers:
+    name: "Tests [MySQL5.7][Py3.7][providers][kerberos]"
+    runs-on: ubuntu-latest
+    needs: [statics, statics-tests]
+    env:
+      TRAVIS_JOB_NAME: "Tests [MySQL5.7][Py3.7][providers][kerberos]"
+      BACKEND: mysql
+      PYTHON_VERSION: 3.7
+      MYSQL_VERSION: 5.7
+      ENABLED_INTEGRATIONS: "kerberos"
+    steps:
+      - uses: actions/checkout@master
+      - name: "Tests [MySQL5.7][Py3.7][providers][kerberos]"
+        run: ./scripts/ci/ci_run_airflow_testing.sh tests/providers
+
+  tests-p37-mysql-kerberos:
+    name: "Tests [MySQL5.7][Py3.7][core][kerberos]"
+    runs-on: ubuntu-latest
+    needs: [statics, statics-tests]
+    env:
+      TRAVIS_JOB_NAME: "Tests [MySQL5.7][Py3.7][core][kerberos]"
+      BACKEND: mysql
+      PYTHON_VERSION: 3.7
+      MYSQL_VERSION: 5.7
+      ENABLED_INTEGRATIONS: "kerberos"
+    steps:
+      - uses: actions/checkout@master
+      - name: "Tests [MySQL5.7][Py3.7][core][kerberos]"
+        run: ./scripts/ci/ci_run_airflow_testing.sh --ignore=tests/providers
+
+  requirements:
+    name: "Generate requirements"
+    runs-on: ubuntu-latest
+    needs: [statics, statics-tests]
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
```

Comment:
Maybe we can make the [3.6, 3.7] into variable as well? That would be great if we could :)

Reply:
The best option would be to extract this logic to yaml template but Github Actions doesn't support it yet...